src/test/java/edu/kis/vh/nursery/RhymersDemo.java

```diff
@@ -6,7 +6,7 @@
 class RhymersDemo {
 
     public static void main(String[] args) {
-        Rhymersfactory factory = new DefaultRhymersFactory();
+        final Rhymersfactory factory = new DefaultRhymersFactory();
```

Comment:
You don't need to put final before here, because it's not a class attribute but a variable inside a function.